<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,7">
       <item>
        <widget class="QFrame" name="frame_botones">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,2,0,0">
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string>Añadir mecanico</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>gotit-icons-r560/icons/edit_normal_hover.png</normaloff>gotit-icons-r560/icons/edit_normal_hover.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Añadir usuario</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>gotit-icons-r560/icons/edit_small_normal_hover.png</normaloff>gotit-icons-r560/icons/edit_small_normal_hover.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Listado de usuarios</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>gotit-icons-r560/icons/license_normal_hover.png</normaloff>gotit-icons-r560/icons/license_normal_hover.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Gestion de usuarios</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>gotit-icons-r560/icons/admin_normal_hover.png</normaloff>gotit-icons-r560/icons/admin_normal_hover.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Gestion de mecanicos</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>gotit-icons-r560/icons/advanced_normal_hover.png</normaloff>gotit-icons-r560/icons/advanced_normal_hover.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Gestion de claves</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>gotit-icons-r560/icons/password_normal_hover.png</normaloff>gotit-icons-r560/icons/password_normal_hover.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_lista">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <widget class="QWidget" name="page_gestion_claves">
             <layout class="QVBoxLayout" name="verticalLayout_4"/>
            </widget>
            <widget class="QWidget" name="page_add_user"/>
            <widget class="QWidget" name="page_list_user"/>
            <widget class="QWidget" name="page_gestion_user"/>
            <widget class="QWidget" name="page_add_mecanic"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
